.github/workflows/standard-tests.yml

```diff
@@ -0,0 +1,112 @@
+        with:
+          java-version: 1.8
+      - run: mvn clean install --projects janusgraph-all --also-make 
+          -DskipTests=true -Dmaven.javadoc.skip=true --batch-mode
+
+  test-matrix:
+    runs-on: ubuntu-20.04
+    needs: build-all
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - module: driver
+          - module: server
+          - module: test
+          - module: inmemory
+            args: "-Dtest.skip.tp=false"
+          - module: berkeleyje
+          - module: lucene
+          - module: solr
+            args: "-Pdocker,solr7"
+          - module: solr
+            args: "-Pdocker,solr8"
+          - module: es
+            args: "-Pelasticsearch7"
+          - module: es
+            args: "-Pelasticsearch6"
+          - module: es
+            args: "-Pelasticsearch60"
+          - module: cql
+            args: "-Pcassandra3-byteordered -Dtest=\"**/diskstorage/cql/*\""
+          - module: cql
+            args: "-Pcassandra3-murmur -Dtest=\"**/diskstorage/cql/*\""
+          - module: cql
+            args: "-Pcassandra3-byteordered -Dtest=\"**/graphdb/cql/*\""
+          - module: cql
+            args: "-Pcassandra3-murmur -Dtest=\"**/graphdb/cql/*\""
+          - module: cql
+            args: "-Pcassandra3-murmur -Dtest=\"**/hadoop/*\""
+          - module: cql
+            args: "-Pcassandra3-murmur-ssl -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
+          - module: cql
+            args: "-Pcassandra3-murmur-client-auth -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
+          - module: hbase-parent/janusgraph-hbase-10
+            args: "-Dtest=\"**/diskstorage/hbase/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            args: "-Dtest=\"**/graphdb/hbase/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            args: "-Dtest=\"**/hadoop/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            install-args: "-Dhbase.profile -Phbase2"
+            args: "-Dtest=\"**/diskstorage/hbase/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            install-args: "-Dhbase.profile -Phbase2"
+            args: "-Dtest=\"**/graphdb/hbase/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            install-args: "-Dhbase.profile -Phbase2"
+            args: "-Dtest=\"**/hadoop/*\""
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - run: mvn clean install --projects janusgraph-${{ matrix.module }} --also-make 
+          -DskipTests=true -Dmaven.javadoc.skip=true --batch-mode ${{ matrix.install-args }}
+      - run: mvn verify --projects janusgraph-${{ matrix.module }} ${{ matrix.args }}
```

Comment:
You removed the `Pcoverage` parameter here if I'm not mistaken. Don't we need that for _codecov_: https://codecov.io/gh/JanusGraph/janusgraph ?

Reply:
changed